templates/packer/ansible/roles/common/files/monit_scripts/verify_worker_connection

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
-# Verifies that a composer worker can reach its composer server.
-
-# Get the hostname of the composer instance.
-COMPOSER_HOST=$(grep -Eo "[a-z0-9\.]+.composer.[a-z0-9\.]+" /etc/hosts)
-
-# Exit now if this is not a worker.
-if [[ -z "${COMPOSER_HOST}" ]]; then
-  echo "Not a worker. Skipping check."
-  exit 2
-fi
-
-# Test a connection to composer.
-CONNECTION_TEST=$(
-  curl -s --connect-timeout 5 \
```

Comment:
so theoretically this would be pretty easy, we could just curl api.(stage.)openshift.com/....composer/openapi every now and then. No auth required, if that's up the service is up.

Reply:
Could it be enough to simply make sure we log sufficiently when the worker fails?

Reply:
Yea, that already happens, /jobs request will log `Requesting job failed` (lvl error).